corehq/blobs/tests/test_migratingdb.py

```diff
@@ -20,17 +20,16 @@ class TestMigratingBlobDB(get_base_class()):
 
     @classmethod
     def setUpClass(cls):
+        super(TestMigratingBlobDB, cls).setUpClass()
+
         with trap_extra_setup(AttributeError, msg="S3_BLOB_DB_SETTINGS not configured"):
             config = settings.S3_BLOB_DB_SETTINGS
+
+        # these are all closed in the base class
         cls.s3db = TemporaryS3BlobDB(config)
         cls.fsdb = TemporaryFilesystemBlobDB()
         cls.db = mod.MigratingBlobDB(cls.s3db, cls.fsdb)
```

Comment:
Looks like this overwrites [an attribute created by `super...setupClass()`](https://github.com/dimagi/commcare-hq/blob/2553af0662c033080df4fd72083222836992a921/corehq/blobs/tests/test_s3db.py#L90) before that db has been cleaned up properly. I think it was intentional that this method did not call the super method because of that. However, you can get around that by replacing `cls.s3db = TemporaryS3BlobDB(config)` with `cls.s3db = self.db`, and can also remove the `with trap_extra_setup...` block.